src/circuit/gadgets/constraints.rs

```diff
@@ -671,7 +646,7 @@ pub fn enforce_n_bits<F: LurkField, CS: ConstraintSystem<F>>(
     n: usize,
 ) -> Result<(), SynthesisError> {
     let num_bits = num.to_bits_le(&mut cs.namespace(|| "u64 remainder bit decomp"))?;
-    let v = num_bits[(n + 1)..255].to_vec();
+    let v = num_bits[n..255].to_vec();
     boolean_summation(&mut cs.namespace(|| "add all MSBs"), &v, &g.false_num);
     Ok(())
```

Comment:
Contrary to what I commented elsewhere, it seems that this function may in fact eforce that `num` is *at most* `n` bits. For example, it looks like this will always be true if `n` is 255.